common/src/main/java/com/cinemamod/mcef/CefUtil.java

```diff
@@ -87,4 +87,46 @@ static CefApp getCefApp() {
     static CefClient getCefClient() {
         return cefClientInstance;
     }
+
+    public static String mimeFromExtension(String ext) {
+//        ext = ext.toLowerCase();
+//        String ret = mimeTypeMap.get(ext);
+//        if (ret != null)
+//            return ret;
+
+        //If the mimeTypeMap couldn't be loaded, fall back to common things
+        switch (ext) {
+            case "htm":
+            case "html":
+                return "text/html";
+
+            case "css":
+                return "text/css";
+
+            case "js":
+                return "text/javascript";
+
+            case "png":
+                return "image/png";
+
+            case "jpg":
+            case "jpeg":
+                return "image/jpeg";
+
+            case "gif":
+                return "image/gif";
+
+            case "svg":
+                return "image/svg+xml";
+
+            case "xml":
+                return "text/xml";
+
+            case "txt":
+                return "text/plain";
+
```

Comment:
Probably add some mime types for video formats?

Reply:
See: [the above](https://github.com/CinemaMod/mcef/pull/55#discussion_r1349774671)
Those video mime types were likely included in that map, which was loaded from a file

Reply:
did this, though I would still like to port the map as well, granted that's not a task for right now